packages/server/graphql/mutations/helpers/updateReflectionLocation/removeReflectionFromGroup.ts

```diff
@@ -12,10 +12,30 @@ const removeReflectionFromGroup = async (reflectionId, {dataLoader}) => {
     .run()
   if (!reflection) throw new Error('Reflection not found')
   const {reflectionGroupId: oldReflectionGroupId, meetingId, promptId} = reflection
+  const oldReflectionGroup = await r
+    .table('RetroReflectionGroup')
+    .get(oldReflectionGroupId)
+    .run()
+  const {sortOrder: oldSortOrder} = oldReflectionGroup
+  const reflectionGroups = await r
+    .table('RetroReflectionGroup')
+    .getAll(meetingId, {index: 'meetingId'})
+    .filter({isActive: true})
+    .orderBy('sortOrder')
+    .run()
   const meeting = await dataLoader.get('newMeetings').load(meetingId)
 
   // RESOLUTION
-  const reflectionGroup = new ReflectionGroup({meetingId, promptId})
+  const nextReflectionGroupIndex =
+    reflectionGroups.findIndex(({id}) => id === oldReflectionGroup.id) + 1
+  let newSortOrder
+  if (nextReflectionGroupIndex >= reflectionGroups.length) {
+    newSortOrder = oldSortOrder + 1
+  } else {
+    const nextSortOrder = reflectionGroups[nextReflectionGroupIndex].sortOrder
+    newSortOrder = (oldSortOrder + nextSortOrder) / 2
+  }
+  const reflectionGroup = new ReflectionGroup({meetingId, promptId, sortOrder: newSortOrder})
   await r
```

Comment:
could we combine a few of these await clauses? less awaiting means less of a chance of race conditions